packages/create-app/templates/default-app/packages/app-next/package.json.hbs

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "app",
+  "version": "0.0.0",
+  "private": true,
+  "bundled": true,
+  "backstage": {
+    "role": "frontend"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "test": "backstage-cli package test",
+    "lint": "backstage-cli package lint"
+  },
+  "dependencies": {
+    "@backstage/cli": "^{{version '@backstage/cli'}}",
+    "@backstage/frontend-defaults": "^{{version '@backstage/frontend-defaults'}}",
+    "@backstage/plugin-app-visualizer": "^{{version '@backstage/plugin-app-visualizer'}}",
+    "@backstage/plugin-catalog": "^{{version '@backstage/plugin-catalog'}}",
+    "@backstage/plugin-home": "^{{version '@backstage/plugin-home'}}",
+    "@backstage/plugin-kubernetes": "^{{version '@backstage/plugin-kubernetes'}}",
+    "@backstage/plugin-scaffolder": "^{{version '@backstage/plugin-scaffolder'}}",
+    "@backstage/plugin-search": "^{{version '@backstage/plugin-search'}}",
```

Comment:
I don't see TechDocs here?

Reply:
Also I think Notifications too.

Reply:
We don't yet ship Notifications by default but I've added Canon and the API Docs, Catalog Graph, Catalog Import, Org, and TechDocs (and its Addons) plugins as those ship by default and have new frontend system support. 🚀